app/src/main/java/com/microsoft/office365/snippetapp/helpers/BaseUserStory.java

```diff
@@ -63,6 +64,18 @@ public String getStringResource(int resourceToGet) {
                 .getString(resourceToGet);
     }
 
+    public String BaseExceptionFormatter(Exception exception, String storyDescription) {
```

Comment:
Suggest "protected" since outside classes unlikely to ever call this. Rename to formatException()? Not sure it really needs to be called "base".